src/App.tsx

```diff
@@ -1,10 +1,14 @@
+import React from 'react';
+import Footer from './components/Footer/Footer'
+
+
+
 function App() {
   return (
     <div>
-      <h1>
-        What to watch
-      </h1>
+      <Footer />
     </div>
+
```

Comment:
it seems div isn't needed here
```
return (
  <Footer />
)
```
should work correctly

Reply:
fixed